frb_example/pure_dart/rust/src/api.rs

```diff
@@ -689,6 +689,39 @@ pub fn is_app_embedded(app_settings: ApplicationSettings) -> bool {
     matches!(app_settings.mode, ApplicationMode::Embedded)
 }
 
+// use a stream of a mirrored type
+pub fn app_settings_stream(sink: StreamSink<ApplicationSettings>) {
```

Comment:
nit: oops I forgot to mention this: please add a corresponding test in .dart, as the checklist of the PR says:

> New tests are added to ensure new features are working. End-to-end tests are usually in the ./frb_example/pure_dart example, more specifically, rust/src/api.rs and dart/lib/main.dart.